mangaki/mangaki/tests/test_merge.py

```diff
@@ -78,10 +93,28 @@ def test_merge_confirmed(self, **kwargs):
             'fields_to_choose': '',
             'fields_required': ''
         }
-        with self.assertNumQueries(36):
+        with self.assertNumQueries(39):
             self.client.post(merge_url, context)
         self.assertEqual(list(Rating.objects.filter(user__in=self.users).values_list('choice', flat=True)), ['favorite'] * 4)
         self.assertEqual(Work.all_objects.filter(redirect__isnull=True).count(), 1)
         self.assertEqual(WorkCluster.objects.count(), 1)
         self.assertEqual(Staff.objects.count(), 2)
+        self.assertEqual(Reference.objects.count(), 2)
         self.assertEqual(Work.objects.get(id=self.work_ids[0]).genre.count(), 2)
+
+    # noinspection PyPep8Naming
+    @patch('redis.StrictRedis', autospec=True, create=True)
+    @patch('redis_lock.Lock', autospec=True, create=True)
```

Comment:
patching a Redis lock and a Redis created from a connection pool should be easier as a util decorator for tests. will add a fixme.